packages/strapi-admin/admin/src/app.js

```diff
@@ -62,6 +62,8 @@ const pluginsToLoad = [];
 
 Object.keys(plugins).forEach(current => {
   const registerPlugin = plugin => {
+    strapi.registerPlugin(plugin);
```

Comment:
I see what you mean. 
In order to enable interactions between plugins we have to introduce some features from the upcoming plugin API and since, we cannot change the load of the plugin I had to tweak the code a bit.
Previously, inside a plugin the strapi.registerPlugin was not used and now I have just using in order to retrieve the plugin's configuration in order to use it at the class level.